app/controllers/dishes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class DishesController < ApplicationController
+    def show
+        @dish = Dish.find(params[:id])
+    end
+
+    def update
+        @dish = Dish.find(params[:id])
```

Comment:
The content of this controller action is correct, but typically we want to reserve this controller action for when we are updating a dish's attributes, like its name, description, etc. Here, we are instead creating a relationship between an existing dish and an existing ingredient - we are creating a record in the joins table. So, a more RESTful design option would be to put this in a `create` action in a joins controller, like the DishIngredientsController you have above!